src/utils.ts

```diff
@@ -176,13 +176,16 @@ function post(
   authorization: string
 ): Promise<string> {
   const dataString = JSON.stringify(data);
+  const packageVersion = version;
 
   const options = {
     method: 'POST',
     headers: {
       'Content-Type': 'application/json',
       'Content-Length': dataString.length,
-      Authorization: authorization
+      Authorization: authorization,
+      'supergood-api-type': 'supergood-js',
```

Comment:
this doesnt seem to be the same as the other libraries. Others expecting:
`supergood-api'

Reply:
Fixing to `supergood-api`